gst/nnstreamer/tensor_demux/gsttensordemux.c

```diff
@@ -582,7 +582,7 @@ gst_tensor_demux_get_property (GObject * object, guint prop_id,
       strings = (gchar **) g_ptr_array_free (arr, FALSE);
       p = g_strjoinv (",", strings);
       g_free (strings);
```

Comment:
let's fix all the free function line also (if appended with g_ptr_array_add, it needs to be freed with g_strfreev)
g_strfreev (strings);

Reply:
updated.